dds/DCPS/DataReaderImpl_T.h

```diff
@@ -1476,196 +1521,297 @@ void store_instance_data(
 
-      { ACE_GUARD(ACE_Recursive_Thread_Mutex, instance_guard, this->instances_lock_);
-        int ret = OpenDDS::DCPS::bind(instances_, handle, instance);
+    { ACE_GUARD(ACE_Recursive_Thread_Mutex, instance_guard, this->instances_lock_);
+      int ret = OpenDDS::DCPS::bind(instances_, handle, instance);
 
-        if (ret != 0)
-          {
-            ACE_ERROR ((LM_ERROR,
-                        ACE_TEXT("(%P|%t) ")
-                        ACE_TEXT("%CDataReaderImpl::")
-                        ACE_TEXT("store_instance_data, ")
-                        ACE_TEXT("insert handle failed. \n"), TraitsType::type_name()));
-            return;
-          }
+      if (ret != 0)
+      {
+        ACE_ERROR ((LM_ERROR,
+                    ACE_TEXT("(%P|%t) ")
+                    ACE_TEXT("%CDataReaderImpl::")
+                    ACE_TEXT("store_instance_data, ")
+                    ACE_TEXT("insert handle failed. \n"), TraitsType::type_name()));
+        return;
       }
+    }
 
 #ifndef OPENDDS_NO_OWNERSHIP_KIND_EXCLUSIVE
-      if (this->is_exclusive_ownership_) {
-        if (inst == 0) {
-          inst = new InstanceMap ();
-          this->owner_manager_->set_instance_map(
-                                                 this->topic_servant_->type_name(), reinterpret_cast <void* > (inst), this);
-        }
-
-        if (new_handle) {
-          std::pair<typename InstanceMap::iterator, bool> bpair =
-            inst->insert(typename InstanceMap::value_type(*instance_data,
-                                                 handle));
-          if (bpair.second == false)
-            {
-              ACE_ERROR ((LM_ERROR,
-                          ACE_TEXT("(%P|%t) ")
-                          ACE_TEXT("%CDataReaderImpl::")
-                          ACE_TEXT("store_instance_data, ")
-                          ACE_TEXT("insert to participant scope %s failed. \n"), TraitsType::type_name(), TraitsType::type_name()));
-              return;
-            }
-        }
+    if (this->is_exclusive_ownership_) {
+      if (inst == 0) {
+        inst = new InstanceMap ();
+        this->owner_manager_->set_instance_map(
+          this->topic_servant_->type_name(), reinterpret_cast <void* > (inst), this);
+      }
 
-        if (this->owner_manager_->instance_lock_release () != 0) {
+      if (new_handle) {
+        std::pair<typename InstanceMap::iterator, bool> bpair =
+          inst->insert(typename InstanceMap::value_type(*instance_data,
+            handle));
+        if (bpair.second == false)
+        {
           ACE_ERROR ((LM_ERROR,
                       ACE_TEXT("(%P|%t) ")
                       ACE_TEXT("%CDataReaderImpl::")
                       ACE_TEXT("store_instance_data, ")
-                      ACE_TEXT("release instance_lock failed. \n"), TraitsType::type_name()));
+                      ACE_TEXT("insert to participant scope %s failed. \n"), TraitsType::type_name(), TraitsType::type_name()));
```

Comment:
I can change this, but this code was only re-alligned.  The diff logic is matching the very similar ACE_ERROR messages.